src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/AcceptApprenticeshipUpdates/AcceptApprenticeshipUpdatesCommandHandler.cs

```diff
@@ -55,6 +55,11 @@ protected override async Task Handle(AcceptApprenticeshipUpdatesCommand command,
             await CheckUlnOverlap(command, cancellationToken, apprenticeship, apprenticeshipUpdate);
             if (apprenticeshipUpdate.Email != null)
             {
+                if (apprenticeship.ApprenticeshipConfirmationStatus != null)
```

Comment:
Can we use EmailAddressConfirmedByApprentice instead of ApprenticeshipConfirmationStatus, 
Probably a bit more readable. But just a suggestion.